### ISSUE TITLE
pass additional options to cloudformation

### ISSUE THREAD

**Issue by dmeadows** [bazaarvoice/cloudformation-ruby-dsl#64]
Any additional options don't appear to be forwarded on to cloudformation during stack creation. We need to be able to pass "disable-rollback" for debugging bootstrapping issues.

**Comment by aboutte** [comment]
We disable rollback with `--disable-rollback`.  For example: 

```./cloudformation.rb cfn-create-stack stack-name --region us-west-2 --capabilities CAPABILITY_IAM --parameters "key=value;key2=value" --disable-rollback --timeout 20```

**Comment by dmeadows** [comment]
@aboutte Are you using version 1.0.4?

**Comment by aboutte** [comment]
oops sorry...did not think about that.  I am running 0.4.7.

**Comment by jonaf** [comment]
Good catch, @dmeadows ! It looks like we just need to do some opt parsing and pass anything we don't specifically account for into the options hash we provide to the Cloudformation client in cfntemplate.rb. This should be relatively easy (parse the extra options, convert them to lowercase and hyphens to underscores, and merge them into the options hash) to do, but it is a bug in 1.0.4.

**Comment by jonaf** [comment]
@dmeadows Can you do me a favor and check out b180666 and let me know if it works for you? Note that some options might have slightly altered names (e.g., `--timeout` is now `--timeout-in-minutes`, as Amazon's Ruby SDK has named it as such: http://docs.aws.amazon.com/sdkforruby/api/Aws/CloudFormation/Client.html#create_stack-instance_method).

**Comment by apogoryelyi** [comment]
Hey @jonaf, I've got following error with this codebase:
```
"/Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/model/shapes.rb:204:in `member': no such member :_c (ArgumentError)\n\tfrom /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/param_builder.rb:25:in `block in structure'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/param_builder.rb:23:in `each_pair'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/param_builder.rb:23:in `structure'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/param_builder.rb:16:in `apply'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/handler.rb:48:in `apply_params'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/handler.rb:42:in `build_request'\n\tfrom /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/query/handler.rb:26:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/plugins/user_agent.rb:12:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/client/plugins/endpoint.rb:34:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/client/plugins/raise_response_errors.rb:14:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/aws-sdk-core/plugins/param_converter.rb:21:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/client/plugins/response_target.rb:21:in `call'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/client/request.rb:70:in `send_request'
from /Library/Ruby/Gems/2.0.0/gems/aws-sdk-core-2.1.23/lib/seahorse/client/base.rb:207:in `block (2 levels) in define_operation_methods'
from /Library/Ruby/Gems/2.0.0/gems/cloudformation-ruby-dsl-1.0.4/lib/cloudformation-ruby-dsl/cfntemplate.rb:280:in `cfn'
from /Library/Ruby/Gems/2.0.0/gems/cloudformation-ruby-dsl-1.0.4/lib/cloudformation-ruby-dsl/cfntemplate.rb:503:in `exec!'
from ./service.rb:284:in `<main>'"
```

**Comment by jonaf** [comment]
hi @apogoryelyi. It looks like that error is coming from the Seahorse library in the AWS Ruby SDK. Did you run `bundle install`?

**Comment by apogoryelyi** [comment]
@jonaf it seems like I do my test incorrectly, please merge this change

**Comment by jonaf** [comment]
Thanks for verifying, @apogoryelyi !

**Comment by jonaf** [comment]
Closed by #68
